src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.DocumentationCommentWalker.cs

```diff
@@ -149,14 +149,53 @@ public override void DefaultVisit(SyntaxNode node)
                         nameSyntax = ((XmlElementStartTagSyntax)node).Name;
                     }
 
-                    if (nameSyntax != null && nameSyntax.Prefix == null &&
-                        DocumentationCommentXmlNames.ElementEquals(nameSyntax.LocalName.ValueText, DocumentationCommentXmlNames.IncludeElementName))
+                    if (nameSyntax != null && nameSyntax.Prefix == null)
                     {
-                        _includeElementNodes.Add((CSharpSyntaxNode)node);
+                        if (DocumentationCommentXmlNames.ElementEquals(nameSyntax.LocalName.ValueText, DocumentationCommentXmlNames.IncludeElementName)
+                            || DocumentationCommentXmlNames.ElementEquals(nameSyntax.LocalName.ValueText, DocumentationCommentXmlNames.InheritdocElementName))
+                        {
+                            _includeElementNodes.Add((CSharpSyntaxNode)node);
+                        }
                     }
                 }
 
                 base.DefaultVisit(node);
+
+                // Local function
+                bool isIgnoredCrefAttribute()
```

Comment:
can you have a return statement above this.  it helps prevent someone from adding code after the local function.

Reply:
Sure, but remember this is a prototype (proof of concept)